source/plugin/offline-userplayer-data.rst

```diff
@@ -0,0 +1,71 @@
+=======================================
+Accessing data of offline Users/Players 
```

Comment:
Well, custom captions just for the index are supported, but I get what you mean->changing